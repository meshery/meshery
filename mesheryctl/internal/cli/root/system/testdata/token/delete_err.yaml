contexts:
  local:
    endpoint: http://localhost:9081
    token: default
    platform: docker
    components:
      - meshery-istio
    channel: stable
    version: v0.5.10
current-context: local
tokens:
  - name: default
    location: auth.json
  - name: default2
    location: ~/.meshery/auth.json
