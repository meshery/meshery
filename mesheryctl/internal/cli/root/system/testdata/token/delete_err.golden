Error: Could not delete token "default2" from config: Unsupported Config Type ""
